scrapy_zyte_api/handler.py

```diff
@@ -111,8 +146,9 @@ async def _download_request(
                 f"Got an error when processing Zyte API request ({request.url}): {er}"
             )
             raise IgnoreRequest()
+        finally:
+            self._update_stats()
 
-        self._stats.inc_value("scrapy-zyte-api/request_count")
```

Comment:
Should we keep this stat?

Reply:
The new equivalent of this would be `scrapy-zyte-api/input_queries` right?

The only concern here would be some users relying on checking this stat. Though I think that could easily be addressed by reflecting it clearly in the Changelog.